lib_nbgl/src/nbgl_use_case.c

```diff
@@ -2431,7 +2434,7 @@ void nbgl_useCaseAddressReview(const char                      *address,
 #ifdef TARGET_STAX
     centeredInfo->text3 = NULL;
 #else   // TARGET_STAX
-    centeredInfo->text3               = "Swipe to review";
+    centeredInfo->text3               = "Swipe to continue";
```

Comment:
Is this wording specific to address review?
Else "Swipe to review" must be change in others places in this file.